docs/deploy.rst

```diff
@@ -96,13 +96,13 @@ example, we have the following:
 .. code-block:: ini
 
     [global]
-    server.socket_host: "0.0.0.0"
-    server.socket_port: 9000
-    tools.proxy.on: True
+    server.socket_host= "0.0.0.0"
```

Comment:
Nitpick, I would add a space on both sides of the `=` since that matches what is currently in our cfg file.